src/pages/WorldsDashboard/WorldsDashboard.scss

```diff
@@ -51,4 +51,21 @@
   &__messages-container {
     text-align: center;
   }
+
+  &__worlds-list {
+    background-color: $dark;
+
+    display: grid;
+    width: 100%;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: max-content;
+    grid-gap: 1rem;
+  }
+
+  &__world-panel {
+    border: 1px solid white;
```

Comment:
```suggestion
    border: 1px solid var(--content--over);
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2371/commits/cef8d195f3cdcfef440c04428ffe7f938bbb620f